### ISSUE TITLE
AuthV2: Missing Role names in API

### ISSUE THREAD

**Issue by kiblik** [DefectDojo/django-DefectDojo#4453]
**Is your feature request related to a problem? Please describe**
API supports to see Member:Product and Member:ProductType relations including Role ID. However, it is not possible to show the list of Role names.

Bonus:
Prefetch roles

**Describe the solution you'd like**
API call `GET /roles/` which will return something like:
```json
{
...
  "results": [
    {
      "id": 0,
      "name": "Reader"
    },
...
  ],
...
}
```

**Describe alternatives you've considered**
As I know, there is not alternative

**Comment by StefanFl** [comment]
Roles are currently not in the database but hardocded. This will be a future enhancement, too make roles configurable and store them in the database. 

I would actually see an alternative: The role name could be given with every member, not as a separate endpoint. Would that work for you?

**Comment by kiblik** [comment]
I suppose this is acceptable for `GET`s - fetching actual setting.
However, it is not usable for `PUT`s, `POST`s and `PATCH`es - creating/updating roles

**Comment by StefanFl** [comment]
Alright, I will implement the names for the `GET`s of the members. 

Creating and updating of roles is currently not possible because they are hardcoded. When we will implement configurable roles, they will get API endpoints.

**Comment by valentijnscholten** [comment]
fixed in dev / 2.0.0
